### PR TITLE
Update to 2.90

### PR DESCRIPTION
**Pull request by barthalion**

### CODE REVIEW

**Comment by barthalion** [comment]
cc @hfiguiere @eszlari 

I don't think it's maintainable in the long term to build it from source unless someone who uses it regularly steps up. I'd rather have it use upstream binary than fall out of date for x86_64. WDYT?

**Comment by barthalion** [comment]
bot, build

File: org.blender.Blender.json [hunk 1]
```diff
@@ -365,8 +50,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://ftp.videolan.org/pub/x264/snapshots/x264-snapshot-20191217-2245-stable.tar.bz2",
-                    "sha256": "b2495c8f2930167d470994b1ce02b0f4bfb24b3317ba36ba7f112e9809264160"
+                    "url": "http://ftp.videolan.org/pub/x264/snapshots/x264-snapshot-20120209-2245-stable.tar.bz2",
+                    "sha256": "22b9daa2322ea9fd5a62454ecaf22feac2db36b6e0aeac5c52d901f56399f2f9"
                 }
```
Comment (alexlarsson) [inline org.blender.Blender.json:55]: Why switch to an 8 year old x264 snapshot?

**Comment by barthalion** [comment]
bot, build

**Comment by hfiguiere** [comment]
> cc @hfiguiere @eszlari
> 
> I don't think it's maintainable in the long term to build it from source unless someone who uses it regularly steps up. I'd rather have it use upstream binary than fall out of date for x86_64. WDYT?

It was my understanding that compiling from source was the policy.

**Comment by barthalion** [comment]
It's preferred but not required if there's a good reason.

**Comment by hfiguiere** [comment]
My take right now is that aarch64 user can be sad since they no longer have a build.

Sorry for the delayed answer.

**Comment by barthalion** [comment]
True, it's non-trivial to get it build from source though. I don't mind if someone submits a PR but it's not going to be me.

**Comment by eszlari** [comment]
Someone could open a request upstream for an official aarch64 build.

**Pull request merged**